### PR TITLE
Feature/servtools 371 move the javascript packages from bitbucket to git hub

### PR DESCRIPTION
**Pull request by AaronTrippaers**
Usage: https://github.com/ExtraHorizon/service-development-tools/pull/1/files
Build: https://github.com/ExtraHorizon/service-development-tools/actions/runs/2794643066
Build And Publish: https://github.com/ExtraHorizon/service-development-tools/runs/7666558083?check_suite_focus=true

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ on:
 jobs:
```
Comment (AaronTrippaers) [inline README.md:17]: Review the whole file, the rest accidentally ended up on the dev branch already

**Comment by AaronTrippaers** [comment]
Usage: https://github.com/ExtraHorizon/service-development-tools/pull/1/files
Build: https://github.com/ExtraHorizon/service-development-tools/actions/runs/2794643066
Build And Publish: https://github.com/ExtraHorizon/service-development-tools/runs/7666558083?check_suite_focus=true

File: .github/workflows/buildAndPublish.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+name: Build and Publish Node Package
+on:
+  workflow_call:
+    inputs:
+      nodeVersion:
+        type: string
+        description: 'Node Version'
+        required: true
```
Comment (wwaaijer-exh) [inline .github/workflows/buildAndPublish.yaml]: Needed for just building a package?

File: .github/workflows/buildAndPublish.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          then
+            branch_tag="dev"
+          else
+            branch_tag="feat"
+          fi
+          sha=${{github.sha}}
+          short_commit_hash=${sha:0:7}
+          image_tag=${{env.PROJECT_VERSION}}-$branch_tag-${{github.run_number}}-$short_commit_hash
+        fi
+        yarn publish --new-version $image_tag --registry https://npm.pkg.github.com/ --verbose --no-git-tag-version
```
Comment (wwaaijer-exh) [inline .github/workflows/buildAndPublish.yaml]: `--registry ...` needed? Not picked up from .npmrc?
`--access restricted` to be safe?

File: .github/workflows/buildAndPublish.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  workflow_call:
+    inputs:
+      nodeVersion:
+        type: string
+        description: 'Node Version'
+        required: true
+    secrets:
+      NPM_TOKEN:
+        required: true
+      GITHUB_PACKAGES_TOKEN:
```
Comment (wwaaijer-exh) [inline .github/workflows/buildAndPublish.yaml]: Read v.s. write token.
Do both need to be required?
Usage of default GITHUB_TOKEN v.s. explicit

File: .github/workflows/buildAndPublish.yaml (hunk 3 above)
Comment (wwaaijer-exh) [inline .github/workflows/buildAndPublish.yaml]: latest vs dev publish?

**Review by wwaaijer-exh — APPROVED** [review]

**Pull request merged**